.github/workflows/cloud-workflow.yml

```diff
@@ -176,13 +176,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Install SwaggerHub CLI
+        run: npm install --global swaggerhub-cli
       - name: Publish SwaggerHub API
+        if: github.ref == 'refs/heads/master'
```

Comment:
For now, this entire job (cloud-upload) should only be run for master -- is it possible to mark "on master" for the job, rather than individual steps? @quentinDERORY 

If yes, let's do that. If no, then @wangyyt1013 don't forget to make the "npm install" also only for master

Reply:
yes you can define it at the job level at the same level as runs-on. 
This exact if statement will work!